source/sass/base/_global.scss

```diff
@@ -0,0 +1,6 @@
+// Global changes
+//////////////////////////////////////////////////////////////////
+
+body {
+  background-color: $gray-100;
```

Comment:
are you sure. could use using customizing of bootstrap ?

Reply:
This is a bootstrap var.